doc/WebSite/Multi-Tenancy.md

```diff
@@ -21,183 +21,182 @@ There are some different multi-tenant database & deployment approaches:
 
 ##### Multiple Deployment - Multiple Database
 
-This is **not multi tenancy** actually. But, if we run **one instance**
+This is **not multi-tenancy** actually, but if we run **one instance**
 of the application **for each** customer (tenant) with a **seperated
-database**, we can serve to **multiple tenants** in a single server. We
-just make sure that multiple instance of the application don't
-**conflict** with each other in same server environment.
+database**, we can serve **multiple tenants** on a single server. We
+just have to make sure that multiple instances of the application don't
+**conflict** with each other on the same server environment.
 
-This can be possible also for an **existing application** which is not
-designed as multitenant. It's easier to create such an application since
-the application has not aware of multitenancy. But there are setup,
+This can also be possibl for an **existing application** which is not
```

Comment:
there is a type here, "possibl" must be "possible"

Reply:
Done